include/superior_mysqlpp/low_level/dbdriver.hpp

```diff
@@ -221,7 +221,7 @@ namespace SuperiorMySqlpp { namespace LowLevel
          * @param value Logger instance.
          */
         template<typename T>
-        void setLoggerPtr(T&& value)
+        void setLoggerPtr(T&& value) noexcept
```

Comment:
I think, that noexcept is not appropriate here, because `loggerPtr` is of type `Loggers::SharedPointer_t` which is `std::shared_ptr`. And because this function is templated, you can pass in e.g. `std::unique_ptr`, and in that case it is not noexcept.

Reply:
ACK